pom.xml

```diff
@@ -1,625 +1,657 @@
-<project xmlns="http//maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
James, looks like you have git end of line issues, can you please fix them so that the diff here only shows your changes?